Tests/XpringClientIntegrationTests.swift

```diff
@@ -0,0 +1,78 @@
+import XCTest
+import XpringKit
+
+/// Integration tests run against a live remote client.
+final class XpringClientIntegrationTests: XCTestCase {
+  private let legacyClient = XpringClient(grpcURL: .legacyRemoteURL, useNewProtocolBuffers: false)
+  private let client = XpringClient(grpcURL: .remoteURL, useNewProtocolBuffers: true)
+
+  // MARK: - rippled Protocol Buffers
+
+  func testGetBalance() {
+    do {
+      _ = try client.getBalance(for: Wallet.testWallet.address)
```

Comment:
Where is `Wallet.testWallet` in scope here? is that from `import XpringKit`?

Reply:
This comes from this file:
https://github.com/xpring-eng/XpringKit/blob/master/Tests/Helpers/Wallet%2BTest.swift

We're using extensions to add a static property onto the `Wallet` object. This lives in the `XpringKitTests` target. This pattern looks weird at first but it's really useful to be able to attach static properties onto classes under test from XpringKit. In fact, we have an entire folder of them: 
https://github.com/xpring-eng/XpringKit/tree/master/Tests/Helpers